src/trousse/feature_fix.py

```diff
@@ -335,7 +325,7 @@ def encode_single_categorical_column(
     else:
         logging.error(
             f"No valid encoding_func argument. Possible "
-            f"values are: {[e.name for e in EncodingFunctions]}"
+            f"values are: {[e.name for e in ['Ordinal', 'Categorical']]}"
```

Comment:
Looking at the options you wrote above for encoding.. I think they should be
```suggestion
            f"values are: {[e.name for e in ['ORDINAL', 'ONEHOT']]}"
```